src/main/java/frc/robot/subsystems/vision/Limelight.java

```diff
@@ -0,0 +1,121 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems.vision;
+
+import frc.robot.subsystems.drive.DrivingDeltas;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+/**
+ * Add your docs here.
+ */
+public class Limelight extends SubsystemBase {
+
+    private final double KpAIM = 0.005; 
+    private final double KpDISTANCE = 0.025; // larger than kpAim in initial
+    private final double DEADBAND_DEGREES = 0;
```

Comment:
A DEADBAND of 0 is no deadband at all.  Set to 0.5 for now.  If this is the same deadband we want for the LimelightTopTarget Command, we might want a constants file so we're not having to change the same variable all over the place.